deployment/elk_pod_template.yaml

```diff
@@ -304,6 +304,6 @@ parameters:
 - description: Storage class to use
   value: ocs-storagecluster-ceph-rbd
   name: STORAGE_CLASS
-- description: Image tag
-  value: devel
-  name: IMAGE_TAG
+- description: LogsViewer image
+  value: quay.io/vladikr/logsviewer:devel
```

Comment:
we should move the default image out of my quay repo, at some point

Reply:
as long as you have the repository in your account I think you should own the default image
but maybe an option is [GitHub Packages](https://github.com/features/packages), which is free for open-source projects, making the image linked to the repository, and we could build and push it there on every release/main branch merge